### PR TITLE
[BugFix, TotalParticipantCount] Fix participant count when alone, fix sample to allow joining existing call

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Fixes issue where if you're alone in a call the count has the unformatted string
Fixes issue where you could not join an existing call in the Calling sample
# Why

image

https://skype.visualstudio.com/SPOOL/_workitems/edit/3373830
# How Tested

Tested locally, added e2e tests for scenario

### CODE REVIEW

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx [hunk 1]
```diff
@@ -56,7 +56,9 @@ export const ParticipantListWithHeading = (props: {
   const subheadingUniqueId = useId();
   const theme = useTheme();
   /* @conditional-compile-remove(total-participant-count) */
-  const totalParticipantCount = participantListProps.totalParticipantCount;
+  const totalParticipantCount = participantListProps.totalParticipantCount
+    ? participantListProps.totalParticipantCount
+    : 1;
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: Why is `totalParticipantCount` undefined if you're the only one in the call? Shouldn't it correctly be 1?

**Review by carocao-msft — APPROVED** [review]

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: Another Q - In `paneTitleTrampoline` below, what happens on stable? Presumably the string is still `In this call {numberOfPeople}` so if we conditionally compile the `formatString` call won't we see in stable builds `In this call {numberOfPeople}`?

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: it comes back from the sdk as undefined when you're alone for some reason, not totally sure why.

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: great question, I believe when I originally did it it was not there as expected but I will take look

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: oh right in stable we are still using the old control bar so it is not used at all 
image

Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: Oh this will be impacted by this active PR then! #3408

Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: Actually callwithchat stable already should use the people pane too

Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: > it comes back from the sdk as undefined when you're alone for some reason, not totally sure why.

Sounds like a bug :L

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: image
yeah seems like it

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: > Actually callwithchat stable already should use the people pane too

good point will see what happens there

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:61]: yup was broken, I think I have a better fix coming

**Review by prabhjot-msft — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**